README.md

```diff
@@ -35,9 +34,7 @@ details.
 The container image can be built to be used as a local development service (for renku_crc):
 `docker build -f projects/renku_crc/Dockerfile . --build-arg DEV_BUILD=true -t renku-crc`
 
-It can then be ran as daemon: `docker run -d -e DUMMY_STORES=true --name
-renku-crc renku-crc`
-
+It can then be run as daemon: `docker run -d -e DUMMY_STORES=true --name renku-crc renku-crc`
```

Comment:
**suggestion:** We should probably rename this to renku-data-service, since renku-crc isn't really a thing anymore